src/components/dropdown-menu/DropdownMenuGroupLabel.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import styled from '@emotion/styled';
+import * as radixDropdownMenu from '@radix-ui/react-dropdown-menu';
+import { PropsWithChildren } from 'react';
+
+export function DropdownMenuGroupLabel({
+  ariaLabel,
+  children,
+}: Readonly<PropsWithChildren<{ ariaLabel?: string }>>) {
+  return <StyledGroupLabel aria-label={ariaLabel}>{children}</StyledGroupLabel>;
+}
+
+const StyledGroupLabel = styled(radixDropdownMenu.Label)`
+  color: var(--echoes-color-text-default);
+  font-size: var(--echoes-font-size-body-small);
+  font-weight: var(--echoes-font-weight-semi-bold);
```

Comment:
Let's use the correct font directly instead of reacreating it with different props. Something like : `font: var(--echoes-typography-paragraph-small-semi-bold);`